articles/quickstart/webapp/aspnet-core-beta/01-login.md

```diff
@@ -12,29 +12,31 @@ github:
 contentType: tutorial
 useCase: quickstart
 ---
+
 <!-- markdownlint-disable MD041 -->
 
-<%= include('../../../_includes/_new_app', { showClientSecret: false, isPublicClient: false }) %>
+<%= include('../../../\_includes/\_new_app', { showClientSecret: false, isPublicClient: false }) %>
```

Comment:
```suggestion
<%= include('../../../\_includes/\_new\_app', { showClientSecret: false, isPublicClient: false }) %>
```

Reply:
I reverted these changes, I had no intention to change anything regarding escaping underscores. This must have happened automatically.

Reply:
This happened to me too when I was updating PHP's quickstart docs using VSCode's markdown formatter awhile back; seems to aggressively and unnecessarily escape things.